roachpb/batch.go

```diff
@@ -62,8 +62,13 @@ func (ba *BatchRequest) IsRange() bool {
 
 // GetArg returns the first request of the given type, if possible.
```

Comment:
comment needs an update since it'll look for the "last" `EndTransaction` (there should only ever be at most one and if so at that position, but still).